charts/vela-core/README.md

```diff
@@ -153,6 +153,8 @@ helm install --create-namespace -n vela-system kubevela kubevela/vela-core --wai
 | `authentication.withUser`     | Application authentication will impersonate as the request User                                                            | `true`               |
 | `authentication.defaultUser`  | Application authentication will impersonate as the User if no user provided in Application                                 | `kubevela:vela-core` |
 | `authentication.groupPattern` | Application authentication will impersonate as the request Group that matches the pattern                                  | `kubevela:*`         |
+| `sharding.enabled`            | Enable sharding for core controller                                                                                        | `false`              |
```

Comment:
should the default value to be true?

Reply:
Currently no. If the default value set to be true, then when no shards added, application will not be handled.

Reply:
Behaviour changed. Master shard by default accept applications now.